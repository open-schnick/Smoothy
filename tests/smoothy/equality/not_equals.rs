use smoothy::{assert_that, EqualityAssertion};

mod succeeds {
    use super::*;

    #[test]
    fn with_numbers() {
        assert_that(12).not_equals(21);
    }

    #[test]
    fn with_strings() {
        assert_that(String::from("Yo")).not_equals("Hello There!");
    }
}

mod fails {
    use super::*;
    use crate::failing_assertion;

    #[test]
    fn with_matching_values() {
        failing_assertion!({
            assert_that(12).not_equals(12);
        });
    }
}
