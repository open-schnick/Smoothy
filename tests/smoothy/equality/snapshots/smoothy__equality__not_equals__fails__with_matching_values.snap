---
source: tests/smoothy/equality/not_equals.rs
expression: assertion_failed_output
---
assertion failed: `(left != right)`

Both sides:
12
